lexibank_bowernpny.py

```diff
@@ -12,16 +12,10 @@
 from pylexibank.dataset import Dataset as BaseDataset, Language as BaseLanguage
 from pylexibank.util import getEvoBibAsBibtex
 
-
-@attr.s
-class Language(BaseLanguage):
-    Glottolog_name = attr.ib(default=None)
-
-
 class Dataset(BaseDataset):
     dir = Path(__file__).parent
     id = 'bowernpny'
-    language_class = Language
+    language_class = BaseLanguage
```

Comment:
This line isn't necessary, because that's the default anyway.

Reply:
Oh, should have checked that. I'll change it.